### ISSUE TITLE
Работа одной службы по протоколу яндекса и мейла.

### ISSUE THREAD

**Issue by gegemon2012** [yar229/WebDavMailRuCloud#191]
Здравствуйте!
Как я понял одна запущенная служба может работать сразу с одним или несколькими сетевыми дисками mail или yandex?
А вот работать одновременно с сетевыми дисками mail и yandex одна служба не может и надо запускать две отдельные службы для mail и для yandex из разных каталогов с разными открытыми портами?

**Comment by yar229** [comment]
Да, именно так, для мэйла и яндекса надо запустить разные экземпляры.
В планах есть сделать возможность запуска одним экземпляром, но по срокам ничего пока сказать не могу.

**Comment by gegemon2012** [comment]
У меня вопрос возник по настройке одновременного подключения яндекса и мейла. Я установил службу с портом 801 для мейла.
Создал вторую папку  WebDAVCloudYandRu с аналогичных содержимым для яндекса. Сделал bat файл следующего содержания:
"C:\Program files\WebDAVCloudYandRu\wdmrc.exe" --protocol YadWeb --port 802 --install WDMRC
Pause
Запускаю - пишет указанная служба уже существует.
Что я делаю неправильно?

**Comment by yar229** [comment]
Вторую папку делать не надо, можно из одной.

Бат-файл делать не надо, ключ `--install` создает службу, которая сама будет запускаться, достаточно просто выполнить эту команду и забыть про неё.

`--install WDMRC` - указывает, с каким именем сделать службу. Поскольку для мейла вы уже сделали службу с таким именем, то совершенно закономерно выдается эта ошибка.

Т.е. для яндекса выполните с правами администратора команду
`"C:\Program files\WebDAVCloudYandRu\wdmrc.exe" --protocol YadWeb --port 802 --install WDYAD` - т.е. задав другое имя службы.

После устновки обеих служб либо перезагрузитесь, либо выполните команды запуска обеих служб
```
net start WDMRC
net start WDYAD
```

**Comment by rediffusion** [comment]
Hello! 🙂
HELP! I got errors!

 This is my .bat file:
```
"%~dp0WebDAV_yandex\wdmrc.exe" --protocol YadWeb --port 804 --install WDYAD
pause
```
```
RUN.bat file:
net use N: /DELETE
net use N: http://127.0.0.1:804 "blablalb" /USER:rocket@yandex.ru /persistent:no
exit
```
---

**There is an error.**

```
C:\Users\SAS\Desktop\Downloads\Временно!\Cloud Storage\WebDavMailRuCloud\webdavmailru>net use N: /DELETE
Не удалось найти сетевое подключение.

Для вызова дополнительной справки наберите NET HELPMSG 2250.


C:\Users\SAS\Desktop\Downloads\Временно!\Cloud Storage\WebDavMailRuCloud\webdavmailru>net use N: http://127.0.0.1:804 "2rediffusion@yandex.ru /persistent:no
Системная ошибка 1244.

Запрошенная операция не была выполнена, так как пользователь не зарегистрирован.


C:\Users\SAS\Desktop\Downloads\Временно!\Cloud Storage\WebDavMailRuCloud\webdavmailru>pause
Для продолжения нажмите любую клавишу . . .
```
